### PR TITLE
add synonyms to 'autosomal recessive limb-girdle muscular dystrophy t…

### LINKED ISSUE #7220: Beta-sarcoglycan-related limb-girdle muscular dystrophy R4
### Mondo term

autosomal recessive limb-girdle muscular dystrophy type 2E

### Synonyms

Beta-sarcoglycan-related limb-girdle muscular dystrophy R4, Muscular dystrophy, limb-girdle, autosomal recessive 4

### Synonym type

exact

### Additional comments

Please see additional synonyms in Orphanet https://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=EN&Expert=119 and OMIM https://omim.org/entry/604286
Thank you, Gioconda

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
…ype 2E'

close #7220

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -244236,9 +244236,15 @@ subset: ordo_disease {source="Orphanet:119"}
...
+synonym: "beta-sarcoglycan limb-girdle muscular dystrophy" RELATED [GARD:0003851]
+synonym: "beta-sarcoglycan-related LGMD R4" EXACT [Orphanet:119]
+synonym: "beta-sarcoglycan-related limb-girdle muscular dystrophy R4" EXACT [Orphanet:119]
 synonym: "beta-sarcoglycanopathy" BROAD [DOID:0110279, Orphanet:119]
+synonym: "LGMD due to beta-sarcoglycan deficiency" EXACT [Orphanet:119]
+synonym: "LGMD type 2E" EXACT [Orphanet:119]
 synonym: "LGMD2E" EXACT ABBREVIATION [DOID:0110279, MONDO:Lexical, OMIM:604286, Orphanet:119]
+synonym: "LGMDR4" EXACT ABBREVIATION [OMIM:604286]
 synonym: "limb-girdle muscular dystrophy due to beta-sarcoglycan deficiency" EXACT [DOID:0110279, Orphanet:119]
 synonym: "limb-girdle muscular dystrophy type 2E" RELATED [GARD:0003851]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:244249]: This is also a synonym in Orphanet. Should `Orphanet:119` be added and this changed to `EXACT`?

**Review by twhetzel — COMMENTED** [review]
This requested synonym is missing `Muscular dystrophy, limb-girdle, autosomal recessive 4`

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:244249]: will do

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:244249]: done

**Comment by nicolevasilevsky** [comment]
ready for re-review @twhetzel

**Review by twhetzel — APPROVED** [review]
lgtm

**Comment by twhetzel** [comment]
@nicolevasilevsky I can merge this once QC checks finish

**Comment by nicolevasilevsky** [comment]
thank you!

**Pull request merged**